beginner-challenges/challenge-1/index.js

```diff
@@ -0,0 +1,30 @@
+
+//below function will gerrate the array with the userinput
+function generateArray(num){
+    let Array =[];
```

Comment:
It is common practice to use camelCase in JavaScript. Also, 'array' has a concrete meaning in programming and using 'array' can be confusing. It is verbose and in this small example not that relevant but using a variable name that is fitting to the context is preferable.

Another point to think about: You are declaring the array once, therefore, you can use const to declare the variable.

**Example**

``` javascript
// instead of
let Array = []
// use this
const generatedArray=[]
```

Reply:
done